src/openrct2/rct12/RCT12.cpp

```diff
@@ -79,8 +79,7 @@ uint8_t RCT12SurfaceElement::GetSlope() const
 uint32_t RCT12SurfaceElement::GetSurfaceStyle() const
 {
     uint32_t retVal = (terrain >> 5) & 7;
-    if (type & 1)
-        retVal |= (1 << 3);
+    retVal |= (type & RCT12_SURFACE_ELEMENT_TYPE_SURFACE_MASK) << 3;
```

Comment:
Will this have issues with #13536. I assume not as this is only changing the RCT12 element and not the in memory element.

Reply:
It should not affect #13536, indeed.